cmd/server.go

```diff
@@ -78,7 +78,9 @@ var (
 				}
 			}
 
-			zap.L().Sync() // flush the logger
+			if err := zap.L().Sync(); err != nil { // flush the logger
+				logger.Fatalw("Could not flush logger", "Err", err)
```

Comment:
Since we are dealing with logger closing, we should not print error using logger. Instead we should do:
```go
fmt.Fprint(os.Stderr, "Could not flush logger", err.Error())
```